docker/bin/gen-ssl-certs.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+#
```

Comment:
I recognize this script, proper attribution and licensing is typically a good idea, but in this case it is okay not to.

Reply:
Yep sorry I  just ran cp $librdkafka_home/tests/gen-ssl-certs.sh. I'll copy/paste the license info from the other librdkafka source files in